### ISSUE TITLE
Critical message when stopping Wazuh-db 

### ISSUE THREAD

**Issue by chemamartinez** [wazuh/wazuh#1433]
Branch 3.7 on development.

It shows a weird critical message when stopping wazuh-db:

```
2018/09/24 08:17:44 ossec-monitord: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
2018/09/24 08:17:44 ossec-logcollector: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
2018/09/24 08:17:45 ossec-remoted: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
2018/09/24 08:17:45 ossec-syscheckd: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
2018/09/24 08:17:45 ossec-analysisd: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
2018/09/24 08:17:45 ossec-execd: INFO: (1314): Shutdown received. Deleting responses.
2018/09/24 08:17:45 ossec-execd: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
2018/09/24 08:17:45 wazuh-db: INFO: (1225): SIGNAL [(15)-(Terminated)] Received. Exit Cleaning...
2018/09/24 08:17:46 wazuh-db: CRITICAL: at run_worker(): wnotify_wait(): Invalid argument
```

**Comment by chemamartinez** [comment]
Solved at #1454.

**Comment by juliansoc** [comment]
experiencing the same issue here
